cmd/admin-router.go

```diff
@@ -63,9 +63,9 @@ func registerAdminRouter(router *mux.Router, enableIAM bool) {
 
 		/// Health operations
 
-		// Performance command - return performance details based on input type
-		adminV1Router.Methods(http.MethodGet).Path("/performance").HandlerFunc(httpTraceAll(adminAPI.PerfInfoHandler)).Queries("perfType", "{perfType:.*}")
 	}
+	// Performance command - return performance details based on input type
+	adminV1Router.Methods(http.MethodGet).Path("/performance").HandlerFunc(httpTraceAll(adminAPI.PerfInfoHandler)).Queries("perfType", "{perfType:.*}")
```

Comment:
While `CPUUtil` and `MemInfo` are useful for gateways, this change also allows `DrivePerf` call for gateway - which is not useful. We need to handle the gateway case inside the handler, probably return `ErrNotImplemented` for `DrivePerf`